### ISSUE TITLE
Allow production files to be tagged as "final" or "production ready"

### ISSUE THREAD

**Issue by pgoussy** [BirkbeckCTP/janeway#3664]
**Is your feature request related to a problem? Please describe.**

Cleaning up and creating a ticket for a conversation from Discord:

When the Review stage is complete and editors formally accept articles and they move into Copyediting, I am often frustrated to find that editors/authors didn't include "final" in a manuscript's Label or filename. In those cases, I need to sift through the last-modified dates to find the most recent file(s) to send to the copyeditor--which assumes that the most recently updated file is the one intended to go into production. But even _that_ isn't a universal solution, because frequently...

- There may be supplemental files (e.g., separate author bio, tables that contain a lot of text, appendices, etc.) that need copyediting as well, but those might _not_ have been revised since initial upload, so I'd need to find them among the earlier files alongside the now-defunct earlier manuscripts.
-  Supplemental tables/appendices may have been uploaded as separate files _and also_ incorporated into the manuscript, meaning I don't need to send the separate files to the copyeditor at all.
- There may be multiple seemingly-duplicate files uploaded around the same time, and I need to figure out if there are any differences between them. (Was it just an accidental double-upload? Was there a last-minute typo fix? Does one version have embedded comments and the other doesn't?)
- An author may use the term "final" in their manuscript submission, but the editors may have made further changes prior to acceptance and uploaded yet another more recent MS _without_ mentioning "final". (Or, perhaps, an author's _initial_ MS filename includes "final" because it was their personal final draft, but it hasn't gone through peer review yet.) In these cases I need to ask the editors to clarify whether to use the "final" MS or the most-recently-uploaded one.

**Describe the solution you'd like**

Here is my multi-step proposal to mitigate most of these issues:

1. Create some kind of flag/tag/check-box (a concrete value, distinct from the free-text Label field) that can be added to a file to identify it as "final" or, perhaps more accurately, "production ready" 
2. Require that at least one file be tagged as "production ready" before an article is allowed to move from Review into Copyediting
3. (optional) On the Accept Article screen (or whatever the editors see before the article leaves Review) add language to alert editors that 
(a) **all** files marked "production ready" will move forward into the production workflow, and 
(b) all files **not** marked as such will **not** move forward (i.e., the implication being that if they aren't "final", the production team can safely ignore them), so 
(c) editors should confirm that they've tagged the final manuscript **plus** all necessary supplemental files that haven't already been embedded in the manuscript--figures, tables, appendices, etc. (maybe also mention something about resolving all comments and tracked changes in the document before marking it as "production ready")

**Comment by ajrbyers** [comment]
Thanks for this Patrick. I think this would be a useful addition that most editors would appreciate. We'll look at schdeduling it in.

**Comment by philippagrand** [comment]
Hi Patrick - thanks for flagging this in Discord. I agree with your solution and think it would be of great help.  I've had several instances recently of the wrong version of the article going to copyediting causing author upset/anger, cost, wasted time.

Andy - is this planned for the new 1.5.1 upgrade?

**Comment by joemull** [comment]
I agree this would be great. @philippagrand this is not in 1.5.1 as we are trying to get that release out the door, with many long-awaited fixes. But I think it's a great candidate for the [1.6 release](https://github.com/orgs/BirkbeckCTP/projects/13) and we will consider adding it if there's room when we kick off that work.
